aclk/schema-wrappers/context_stream.cc

```diff
@@ -0,0 +1,46 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+#include "proto/context/v1/stream.pb.h"
+
+#include "context_stream.h"
+
+struct stop_streaming_ctxs *parse_stop_streaming_ctxs(const char *data, size_t len)
+{
+    context::v1::StopStreamingContexts msg;
+
+    struct stop_streaming_ctxs *res;
+
+    if (!msg.ParseFromArray(data, len))
+        return NULL;
+
+    res = (struct stop_streaming_ctxs *)calloc(1, sizeof(struct stop_streaming_ctxs));
+
+    if (!res)
+        return NULL;
```

Comment:
On the other hand, this says that we do not want to stop everything :thinking: ,  so I have another question, why do not want to stop when memory cannot be allocated?

Reply:
you mean you want to fatal here?

Reply:
Exactly. We were not able to get memory, why are we still proceeding?  I am still reviewing the code, it is possible the answer is in another place, but it is not clear at this point.

Reply:
well i was thinking lets not send the message then... maybe agent itself can continue monitoring... but you are right everywhere else we use `fatal` so we should do that here as well